internal/mode/static/state/resolver/service_resolver_test.go

```diff
@@ -212,22 +218,17 @@ var _ = Describe("ServiceResolver", func() {
 				},
 			}
 
-			endpoints, err := serviceResolver.Resolve(context.TODO(), svc, 80)
```

Comment:
Do we still need the `svc` variable?

Reply:
Yea I suppose we don't, does the change I made look good?